docker-compose-local-chain.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  cc_alice:
+    container_name: cc-alice
+    image: "centrifugeio/centrifuge-chain:${CC_DOCKER_TAG:-parachain-latest}"
+    ports:
+      - "30355:30333"
+      - "9936:9933"
+      - "9946:9944"
+    volumes:
+      - type: bind
+        source: ./res/rococo-local.json
+        target: /chainspec.json
+        read_only: true
+    command: >
+      --chain="${CC_CHAIN_ID:-development-local}" 
```

Comment:
Should we merge naming for the runtime env var? Here we call it `CC_CHAIN_ID`, in the `init.sh` we call it `PARA_CHAIN_SPEC`